### PR TITLE
Scroll to top

### PR DESCRIPTION
**Pull request by Ayush0730**
## What is the change?
Added auto page scrolling button at center bottom. I have changed Jquery from min slim to min as fadeIn and fadeOut functions were not available in slim version

## Related issue?
close: #285 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

https://user-images.githubusercontent.com/68139755/114510376-3e386000-9c54-11eb-9e7d-b18ae01aaf0d.mp4

### CODE REVIEW

**Comment by Ayush0730** [comment]
Should I use latest version of Jquery, i.e 3.6.0 minified ?

**Comment by Pranav016** [comment]
> Should I use latest version of Jquery, i.e 3.6.0 minified ?

yes

File: public/css/styles.css [hunk 1]
```diff
@@ -759,3 +759,13 @@ input[name='perPage'] {
     transition: all 0.1s ease-in-out;
     filter: blur(20px);
 }
+
+/* -------------------------------------------- */
+/* Scroll To Top Button */
+
+.back-to-top {
+    position: fixed;
+    bottom: 25px;
```
Comment (Pranav016) [inline public/css/styles.css:768]: At the absolute bottom, positioning could be better.
You can place it flush in the small section in the footer by decreasing the bottom margin..

image


Also update `px` to relate unit such as `rem em or vh`

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by Ayush0730** [comment]
image

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**